src/TropicalGeometry/hypersurface.jl

```diff
@@ -210,15 +211,11 @@ end
 @doc raw"""
     dual_subdivision(TropH::TropicalHypersurface)
 
-Return the dual subdivision of `TropH`.  Raises an error, if neither it nor the the internal polymake object is cached.
+Return the dual subdivision used to construct `TropH`.  Raises an error, if it is not cached.
```

Comment:
```suggestion
Return the dual subdivision used to construct `TropH`. Raises an error if it is not cached.
```
Ahh, thank you, that makes it clear!